lang/node-mozilla-iot-gateway/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mozilla-iot-gateway
+PKG_VERSION:=0.3.1
+PKG_RELEASE:=1
+
+PKG_REV:=0.3.1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/mozilla-iot/gateway.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_REV)
+PKG_SOURCE_VERSION=$(PKG_REV)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_REV).tar.gz
+PKG_MIRROR_HASH:=9f5e50975fb582f5ce36a976415b26fe62411452af3bbf4aa4c48be62217d32b
+
+PKG_BUILD_DEPENDS:=node/host openzwave
+PKG_NODE_VERSION:=8.10.0
+
+PKG_MAINTAINER:=Marko Ratkaj <marko.ratkaj@sartura.hr>
+PKG_LICENSE:=Custom
```

Comment:
Looks like the license is MPL-2.0?

Reply:
Thank you @jefferyto, i will update. Not sure why travis build failed though, can't really tell from the build log.